client/src/components/DisplayTable/DisplayTable.tsx

```diff
@@ -297,13 +343,14 @@ export class DisplayTable extends React.Component {
     const visible_ordered_col_keys = _.intersection(
       all_ordered_col_keys,
       visible_col_keys
-    );
+    ) as string[];
+
     const csv_string = _.chain(visible_ordered_col_keys)
-      .map((key) => col_configs_with_defaults[key].header)
+      .map((key: string) => col_configs_with_defaults[key].header)
       .thru((header_row) => [header_row])
       .concat(
-        _.map(sorted_filtered_data, (row) =>
-          _.map(visible_ordered_col_keys, (key) =>
+        _.map<any, string[]>(sorted_filtered_data, (row: DisplayTableData) =>
```

Comment:
To be honest, I don't know how this generic (`<any, string[]`) fixed it if someone can enlighten me on that, please do

Reply:
this gets matched against the lodash type declarations (cmd-click on `_.map` in vscode):

```typescript
        map<T, TResult>(collection: T[] | null | undefined, iteratee: ArrayIterator<T, TResult>): TResult[];
        /**
         * @see _.map
         */
        map<T, TResult>(collection: List<T> | null | undefined, iteratee: ListIterator<T, TResult>): TResult[];
        /**
         * @see _.map
         */
        map<T>(collection: Dictionary<T> | NumericDictionary<T> | null | undefined): T[];
        /**
         * @see _.map
         */
        map<T extends object, TResult>(collection: T | null | undefined, iteratee: ObjectIterator<T, TResult>): TResult[];
        /**
         * @see _.map
         */
        map<T, K extends keyof T>(collection: Dictionary<T> | NumericDictionary<T> | null | undefined, iteratee: K): Array<T[K]>;
        /**
         * @see _.map
         */
        map<T>(collection: Dictionary<T> | NumericDictionary<T> | null | undefined, iteratee?: string): any[];
        /**
         * @see _.map
         */
        map<T>(collection: Dictionary<T> | NumericDictionary<T> | null | undefined, iteratee?: object): boolean[];

```

In this case, I think it's matching the first rule, 

```typescript
map<T, TResult>(collection: T[] | null | undefined, iteratee: ArrayIterator<T, TResult>): TResult[];
```

Which means that it's mapping a list of `any` into  a list of `strings`.  It looks like the mapper function is expecting `DisplayTableData` elements, though. So this should really be `_.map<DisplayTableData, string>`. If you try and make that switch, though, it complains that `sorted_filtered_data` isn't the right type, which probably means we need some more annotations in `sorted_filtered_data`'s derivation above.